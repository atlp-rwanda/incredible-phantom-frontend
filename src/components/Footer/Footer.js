import React from 'react';
import './Footer.scss';
export const Footer = () => {
  return <div className="Footer">Phantom &copy; 2021 - 2022</div>;
};
