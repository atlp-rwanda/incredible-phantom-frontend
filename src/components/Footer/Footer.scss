@import '../../styles/fonts';
@import '../../styles/_variables.scss';

.Footer {
  width: 100%;
  height: 5vh;
  text-align: center;
  font-family: $normalFont;
  font-size: 0.8rem;
  padding: 10px 0px 10px 0px;
}
