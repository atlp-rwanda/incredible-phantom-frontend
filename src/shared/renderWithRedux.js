import '@testing-library/jest-dom';
import React, { Suspense } from 'react';
import thunk from 'redux-thunk';
import { Provider } from 'react-redux';
import { render } from '@testing-library/react';
import { MemoryRouter as Router } from 'react-router-dom';
import { applyMiddleware, createStore } from 'redux';
import rootReducer from '../../redux/reducers/rootReducer';
import '../../i18n';
import { I18nextProvider } from 'react-i18next';
import i18n from 'i18next';
const RenderWithRedux = (children, reduxState = {}) => {
  const store = createStore(rootReducer, reduxState, applyMiddleware(thunk));
  return render(
    <Provider store={store}>
      <I18nextProvider i18n={i18n}>
        <Router>{children}</Router>
      </I18nextProvider>
    </Provider>,
  );
};
export default RenderWithRedux;