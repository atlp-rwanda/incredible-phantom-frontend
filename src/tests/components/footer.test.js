import React from 'react';
import renderWithRedux from '../shared/renderWithRedux';
import { Footer } from '../../components/Footer/Footer';

describe('Footer', () => {
  it('should render the Footer', () => {
    renderWithRedux(<Footer />);
  });
});
