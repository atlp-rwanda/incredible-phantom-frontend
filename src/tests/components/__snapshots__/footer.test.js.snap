// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should return the Footer 1`] = `ShallowWrapper {}`;
